Scripts/summaryStatistics/biophysics/density_calculation.py

```diff
@@ -40,7 +40,7 @@ def convert_cellmodeller_orientation_to_radians(cell_dir):
     
```

Comment:
Do we need an exception handling case here to avoid division by zero?

Reply:
Seems like a good idea here.